firmware/controllers/settings.h

```diff
@@ -20,4 +20,4 @@ void setEngineType(int value);
 void readPin(const char *pinName);
 
 void printDateTime();
-void setDateTime(const char * const isoDateTime);
+void setDateTime(char const * const isoDateTime);
```

Comment:
churn for no gain?

Reply:
probably; wanted to keep them consistent w/re: style (`const` qualifier right of keyword; scales better w/ e.g. pointer types like this)